microproxy/protocol/http2.py

```diff
@@ -38,6 +39,8 @@ def decorate(*args):
 
         self.stream = stream
 
+        self.on_pending_events = on_pending_events or on_unhandled_with_type("PENDING_EVENTS")
+        self.on_header_receive = on_header_receive or on_unhandled_with_type("REQUEST_HEADER_RECEIVE")
```

Comment:
Not quite straightforward for me.
Could you describe how this two events work in our Http2Layer?
Is that there is any unhandled circumstance that need these two event?
Thanks!